src/adaptations/device-layer/include/Weave/DeviceLayer/internal/FactoryProvisioning.h

```diff
@@ -97,7 +97,8 @@ enum
     kTag_DevicePrivateKey       = 3,    // [ byte string ] Manufacturer-assigned device key
```

Comment:
DeviceId, DeviceCert and DevicePrivateKey should probably change to ManufacturerDevice<...>.

Also we should probably change the corresponding tag / method / argument names in weave-factory-provisioning-tool.py and WeaveFactoryProvisioningTool.py.  This could be done in another commit, however.